modules/oe_translation_remote/src/RemoteTranslationRequestEntityTrait.php

```diff
@@ -124,6 +124,22 @@ public function setTranslatedData(string $langcode, array $data): TranslationReq
     return $this;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function removeTranslatedData(string $langcode): TranslationRequestRemoteInterface {
```

Comment:
An additional function to get rid of the data of a language, useful when the translation is finished but the status changes back to "in progress" or "failed".